packages/orchestration/src/examples/send-anywhere.contract.js

```diff
@@ -49,27 +56,40 @@ export const contract = async (
   /** @type {(msg: string) => Vow<void>} */
   const log = msg => vowTools.watch(E(logNode).setValue(msg));
 
-  const { makeLocalAccount } = orchestrateAll(sharedFlows, {});
+  // XXX why can't we do all at once?
+  const makeLocalAccount = orchestrate('f1', {}, sharedFlows.makeLocalAccount);
+  const makeNobleAccount = orchestrate('f2', {}, makeNobleAccountFlow);
```

Comment:
We could do:
```js
const { makeLocalAccount, makeNobleAccount } = orchestrateAll(
    {
      makeLocalAccount: sharedFlows.makeLocalAccount,
      makeNobleAccount: makeNobleAccountFlow,
    },
    {},
  );
```

But i'm not sure it's worth it. Should we delete the `XXX` before merging?

Reply:
agree, this `XXX` can go away.

Reply:
`f1` and `f2` were spike-placeholders, presuming the XXX would get resolved by using `orchestrateAll`.
They should become `makeLocalAccount` and `makeNobleAccountFlow` if we're sticking to 2 `orchestrate` calls, right?